service/matching/tasklist/task_completer.go

```diff
@@ -62,11 +65,12 @@ func (e *DomainIsActiveInThisClusterError) Error() string {
 }
 
 var (
-	errWorkflowExecutionInfoIsNil      = errors.New("workflow execution info is nil")
-	errTaskTypeNotSupported            = errors.New("task type not supported")
-	errTaskNotStarted                  = errors.New("task not started")
-	errDomainIsActive                  = &DomainIsActiveInThisClusterError{Message: "domain is active"}
-	historyServiceOperationRetryPolicy = common.CreateTaskCompleterRetryPolicy()
+	errWorkflowExecutionInfoIsNil           = errors.New("workflow execution info is nil")
```

Comment:
urgh, not for this PR, but we probably want to move away from this pattern:

- the unexported errors we can't check against
- I think, generally, exporting a type error is preferable than a static var error (ie 
```
type ErrDomainNotActive struct {
  Message string 
}
```

Reply:
Will refactor in a later PR